packages/react-core/src/components/AboutModal/AboutModalBox.tsx

```diff
@@ -5,16 +5,13 @@ import styles from '@patternfly/react-styles/css/components/AboutModalBox/about-
 export interface AboutModalBoxProps extends React.HTMLProps<HTMLDivElement> {
   /** content rendered inside the AboutModelBox. */
   children: React.ReactNode;
-  /** additional classes added to the AboutModalBox */
-  className?: string;
 }
 
 export const AboutModalBox: React.FunctionComponent<AboutModalBoxProps> = ({
   children,
-  className = '',
   ...props
 }: AboutModalBoxProps) => (
-  <div role="dialog" aria-modal="true" className={css(styles.aboutModalBox, className)} {...props}>
+  <div className={css(styles.aboutModalBox)} {...props}>
```

Comment:
it looks like you used to be able to pass `className` to this component via `<AboutModal className>` - doesn't look like it works anymore. Is that intentional?

Reply:
I mistakenly removed that when i was cleaning up.  I added it back now.